### PR TITLE
Us 106

### PR DESCRIPTION
**Pull request by jdhinkle**
A lot of reformatting on this one from the coding standards, so sorry it looks like there are tons of changes.

See commit 2344890 below for quick overview of changes

### CODE REVIEW

File: src/net/sf/memoranda/Contact.java [hunk 1]
```diff
@@ -37,173 +44,191 @@ public Contact(String fname, String lname, String phoneNumber, String emailAddre
...
 	public Contact(Element element) {
-		
-		Elements children = element.getChildElements();
-		int numelems = element.getChildCount();
-		
-		for(int i=0;i<numelems;i++){
-			if(children.get(i).getLocalName().equalsIgnoreCase("FirstName")){
+
+		final Elements children = element.getChildElements();
+		final int numelems = element.getChildCount();
```
Comment (lknaron) [inline src/net/sf/memoranda/Contact.java]: Line 74: Variable int numelems not in lower camelCase (CG:4). Recommend: numElems

File: src/net/sf/memoranda/ContactManager.java [hunk 2]
```diff
@@ -9,372 +9,435 @@
...
+		}
+
+		// Master contacts hashmap size should be a minimum of 100 and increase
+		// in MASTERLF intervals
+		final int size = root.getChildElements().size();
 		int initHashSize = MASTERLF * ((size / MASTERLF) + 1);
-		if(initHashSize < 100) {
+
+		if (initHashSize < 100) {
 			initHashSize = 100;
```
Comment (lknaron) [inline src/net/sf/memoranda/ContactManager.java]: Lines 302 & 303: Use of literal values (CG:6). Recommend making 100 constant, such as MASTER_MIN.

**Comment by lknaron** [comment]
I performed a code review of this pull request. Looks like you've meticulously gone through and standardized everything. Looks good. I've commented on a few things that might seem nit-picky, but you might want to take a look at them.

**Comment by jdhinkle** [comment]
Not nit-picky at all. I have a tendency to just pound out code without carefully reviewing my style/formatting so I appreciate the feedback. Thats probably why it looks pretty meticulous, part of my focus for this sprint at least is making sure my code is formatted well and I'm not leaving things messy. I'll make these changes and recommit.

**Comment by vbhagat** [comment]
After reviewing the code. Looks like you have taken care of Larry's comments. Good job on the coding style. I think its ready to be merged.

**Pull request merged**